.github/workflows/test-examples-on-label.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: Test examples on label
+
+on:
+  pull_request:
+
+concurrency:
+  cancel-in-progress: true
+  group: >-
+    ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
I might be wrong, but with reusable workflows you can specify many more things in the file itself. Could this block be moved into `.github/workflows/run-examples.yml`?

Reply:
That works! Thanks!